onnxruntime/test/contrib_ops/matmul_4bits_test.cc

```diff
@@ -108,6 +108,7 @@ void RunTest(int64_t M, int64_t N, int64_t K, int64_t block_size, bool has_zerop
   test.AddAttribute<int64_t>("N", N);
   test.AddAttribute<int64_t>("block_size", block_size);
   test.AddAttribute<int64_t>("bits", QBits);
+  test.AddAttribute<int64_t>("accuracy_level", 0);
```

Comment:
Could you please incorporate the `M`, `N`, `K`, and `block_size` used in `onnxruntime/test/contrib_ops/matmul_nbits.cc` into the tests? Additionally, could you add a loop for `accuracy_level` to enable testing of both `CompFp32` and `CompInt8`?

Reply:
CompInt8 introduces dynamic quantization error, so I need to relax the threshold for CompInt8